config/Config-build.in

```diff
@@ -7,6 +7,13 @@
 
 menu "Global build settings"
 
+	config JSON_ADD_IMAGE_INFO
+		bool "Create JSON info files per build image"
+		default BUILDBOT
+		help
+		  The JSON info files contain information about the device and
```

Comment:
Suggestion:

The generated JSON files, named after the base name of image (e.g. "openwrt-ath79-generic-8dev_carambola2..."), will be stored in the same directory as built image files.

Each JSON file contains meta information about the devices supported by a family of images
and the image files belonging to a particular family.

Reply:
Thank you I will update that